### ISSUE TITLE
[TextInputLayout][Accessibility] Support text is not associated with the input field announcement

### ISSUE THREAD

**Issue by tillygajanayake** [material-components/material-components-android#1063]
**Description:** When select the input field on accessibility mode on the support text is not associated with the input field announcement.

**Expected behavior:** The supporting text should be associated with the input field announcement when selected while accessibility mode on.

**Android API version:** Android API version 28

**Material Library version:** 1.1.0

**TalkBack version:** 5.2.0

**Device:** Emulator Pixel 3 

<img src="https://user-images.githubusercontent.com/61483610/75346417-c2b20d00-58f2-11ea-9e37-93c0ea7f21f9.png" width=300>

**Comment by leticiarossi** [comment]
Hi, TalkBack announcements have been improved in https://github.com/material-components/material-components-android/commit/29f3d05fa8f9301dc076ce0801826826b48ae3f4 (in https://github.com/material-components/material-components-android/releases/tag/1.2.0-alpha03) and should be available in our next stabl release.

**Comment by tillygajanayake** [comment]
Do you have an ETA for your next stable release?

**Comment by tillygajanayake** [comment]
> Hi, TalkBack announcements have been improved in [29f3d05](https://github.com/material-components/material-components-android/commit/29f3d05fa8f9301dc076ce0801826826b48ae3f4) (in https://github.com/material-components/material-components-android/releases/tag/1.2.0-alpha03) and should be available in our next stable release.

Thanks.
